docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -184,6 +184,14 @@ The **Share view** option allows you to link the movement and zoom actions of mu
 You might need to reload the dashboard for this feature to work.
 {{< /admonition >}}
 
+#### No map repeating
+
+The **No map repeating** option prevents the base map tiles from repeating horizontally when you pan across the world. This constrains the view to a single instance of the world map and avoids visual confusion when displaying global datasets.
+
+{{< admonition type="note" >}}
+Enabling this option requires the map to reinitialize.
+{{< /admonition >}}
```

Comment:
Though I see there's a precedent for this, I don't think this needs to be in a note.

```suggestion
Enabling this option requires the map to reinitialize.
```

Reply:
Should it just be included in the same paragraph then?

Reply:
That works!